force_wfmanager/central_pane/plot.py

```diff
@@ -166,7 +180,7 @@ def _update_plot_data(self):
         """Set the plot data model to the appropriate arrays so that they
         can be displayed when either X or Y selections have been changed.
         """
-        if self.x is None or self.y is None:
+        if self.x is None or self.y is None or self._data_arrays == []:
```

Comment:
Fixes the issue when update_data_arrays gets called with value_names set, but no available data ready